### PR TITLE
Restructure folders

### LINKED ISSUE #65: Suggestion: Move 'Catalogs' folder inside 'Scenarios' folder
Hi.
We were wondering if the catalogs should move one level below in the folder structure. The rationale beind this, that right now most scenarios contain a line like `<Directory path="../Catalogs/Vehicles" />`, i.e. when resolving all necessary paths starting from the scenario, you need to go up one level in your local folder structure.

This can be annoying, as the scenario is typically the first-level entrypoint for most tools, so it usually already resides on the highest level in the respective working directory. With the current structure, the user of this repo would then either need to put their catalogs outside of the working dir, or they need to adapt every scenario file accordingly.

Maps, as another dependency of the scenarios, are already on the same level as the scenarios, so we are curious why the catalogs are being treated differently.

Thanks for your help!

### PR DESCRIPTION
**Pull request by arauschert**
- now all files only reference files in subdirectories and not parallel directories
- clarified use of logical and concrete scenarios

Closes #65

### CODE REVIEW

**Comment by jdsika** [comment]
I personally do not like folders with capitals as it seems annoying in Linux systems.

Vorschlag:

> Folders and filename are lowercase and words are separated by - (except the _ as separator for the suffix).

**Comment by jdsika** [comment]
Remove "." inside file names

File: README.md [hunk 1]
```diff
@@ -31,18 +31,18 @@ The focus of the here provided scenarios is on securing the planning aspects of
 
 The execution of the concrete scenarios in the open source tools "esmini", a basic OpenSCENARIO player, and "openPASS", a simulation platform for traffic simulation, is described on Windows:
 
-_Note:_ Currently only esmini supports the OpenSCENARIO 1.1 format. For openPASS please use the [release v0.3.2](https://github.com/asam-oss/OSC-ALKS-scenarios/releases/tag/v0.3.2) with scenarios in OpenSCENARIO 1.0 format.
+_Note:_ Currently only esmini supports the OpenSCENARIO 1.3 format. For openPASS please use the [release v0.3.2](https://github.com/asam-oss/OSC-ALKS-scenarios/releases/tag/v0.3.2) with scenarios in OpenSCENARIO 1.0 format.
```
Comment (jdsika) [inline README.md:34]: Is it still the right tag for openPASS?

**Review by jdsika — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (arauschert) [reply README.md:34]: There was no test with a newer version of openPASS, so this sentence is still valid.

**Pull request merged**